app/src/hooks/useTheme.tsx

```diff
@@ -0,0 +1,9 @@
+import useSelector from 'hooks/useSelector';
+import { getTheme } from 'modules/selectors';
+
+
+export default function useTheme() {
+  const theme = useSelector(getTheme)
+  document.documentElement.setAttribute("data-theme", theme);
```

Comment:
Lets have this a hook, so we don't write to DOM on every hook refresh.

```suggestion
  React.useEffect(() => document.documentElement.setAttribute("data-theme", theme), [theme])
```